src/components/landing/Navbar.js

```diff
@@ -0,0 +1,128 @@
+                            <a
+                                href="https://material-tailwind.com?ref=mtk"
+                                target="_blank"
+                                rel="noreferrer"
+                            >
+                                Material Tailwind
+                            </a>
+                        </NavbarBrand>
+                        <NavbarToggler
+                            onClick={() => setOpenNavbar(!openNavbar)}
+                            color="white"
+                        />
+                    </NavbarWrapper>
+
+                    <NavbarCollapse open={openNavbar}>
+                        <Nav>
+                            <div className="flex flex-col z-50 lg:flex-row lg:items-center">
+                                <NavItem
+                                    href="https://material-tailwind.com/documentation/quick-start?ref=mtk"
+                                    target="_blank"
+                                    rel="noreferrer"
+                                    ripple="light"
+                                >
+                                    <Icon name="description" size="2xl" />
+                                    &nbsp;Docs
+                                </NavItem>
+                                <NavItem
+                                    href="https://material-tailwind.com/components?ref=mtk"
+                                    target="_blank"
+                                    rel="noreferrer"
+                                    ripple="light"
+                                >
+                                    <Icon name="apps" size="2xl" />
+                                    &nbsp;Components
+                                </NavItem>
+                                <div className="text-white">
+                                    <Dropdown
+                                        color="transparent"
+                                        size="sm"
+                                        buttonType="link"
+                                        buttonText={
+                                            <div className="py-2.5 font-medium flex items-center">
+                                                <Icon
+                                                    name="view_carousel"
+                                                    size="2xl"
+                                                    color="white"
+                                                />
+                                                <span className="ml-2">
+                                                    Templates
+                                                </span>
+                                            </div>
+                                        }
+                                        ripple="light"
+                                    >
+                                        <Link to="/landing">
```

Comment:
Since this page is the same with the one from `/`, lets keep just one of them, either `/`, either `/landing`